server/app.py

```diff
@@ -19,6 +19,14 @@ class Book(db.Model):
   genre = db.Column(db.String(100), nullable=False)
   star_rating = db.Column(db.String(100), nullable=False)
   book_read = db.Column(db.Boolean)
+  
+  def __init__(self, title, author, url, genre, star_rating, book_read):
+    self.title = title
+    self.author = author
+    self.url = url
+    self.genre = genre
+    self.star_rating = star_rating
+    self.book_read = book_read
```

Comment:
Please remove this __init__

Please refer to this page to learn more.  https://flask-sqlalchemy.palletsprojects.com/en/2.x/quickstart/